core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BpmBillingProjectLifecycle.scala

```diff
@@ -111,7 +112,8 @@ class BpmBillingProjectLifecycle(
           config.azureConfig.get.landingZoneVersion,
           config.azureConfig.get.landingZoneParameters,
           profileModel.getId,
-          ctx
+          ctx,
+          config.azureConfig.get.landingZoneId
```

Comment:
Threading in the optional LZ ID for tests to attach to, if present in the config. It is absent in all environments for now.